frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -907,7 +907,7 @@ class CreateInterface extends React.Component {
                                 </DropdownButton>
```

Comment:
as a consequence of this change, the `roundNavs` array (and associated dropdown items) https://github.com/Ishita-0112/dynabench/blob/4f4a7ace48e58da7848eb9d52eea4a5433007fda/frontends/web/src/common/Annotation/CreateInterface.js#L567-L591 

as well as the `updateSelectedRound` function https://github.com/Ishita-0112/dynabench/blob/4f4a7ace48e58da7848eb9d52eea4a5433007fda/frontends/web/src/common/Annotation/CreateInterface.js#L153-L170 

and the warning displayed when selecting a previous round https://github.com/Ishita-0112/dynabench/blob/4f4a7ace48e58da7848eb9d52eea4a5433007fda/frontends/web/src/common/Annotation/CreateInterface.js#L958-L966

is no longer necessary, so I think they can be safety removed

Reply:
I think this actually also holds true to all the logic involving `task.selected_round` since in this component, selected_round should always === cur_round? eg. no need to even initialise `selected_round`, and you can pass `cur_round` straight to `api.storeExample`

Reply:
Yes true! Thanks!